odoo/addons/base/ir/ir_model.py

```diff
@@ -129,7 +129,7 @@ def _get(self, name):
         """ Return the (sudoed) `ir.model` record with the given name.
         The result may be an empty recordset if the model is not found.
         """
-        return self.sudo().search([('model', '=', name)])
+        return self.sudo().search([('model', '=', name)], order='id')
```

Comment:
Because it crashes when you install the module.
The field `is_mail_thread` does not exist in the database yet, but you need to search for models anyway. Ordering them crashes.
Maybe I should replace this search() by an explicit SQL query...